pootle/settings/50-project.conf

```diff
@@ -38,9 +38,7 @@ MIDDLEWARE_CLASSES = [
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [
-            working_path(os.path.join('templates')),
-        ],
+        'DIRS': [],
```

Comment:
``[]`` is the default.  In this case rather remove it completely.  Please add the reason the the commit message, its missing at the moment.

Is the real reason not that [APP_DIRS](https://docs.djangoproject.com/en/1.9/ref/settings/#app-dirs) is not set in the default config?